pyang/plugins/mef.py

```diff
@@ -0,0 +1,36 @@
+"""MEF usage guidelines plugin
+See MEF Assigned Names and Numbers (MANN) at https://wiki.mef.net/display/MANN/MEF+Assigned+Names+and+Numbers
+"""
+
+import optparse
+import sys
+
+from pyang import plugin
+from pyang import statements
+from pyang import error
+from pyang.error import err_add
+from pyang.plugins import lint
+
+def pyang_plugin_init():
+    plugin.register_plugin(MEFPlugin())
+
+class MEFPlugin(lint.LintPlugin):
+    def __init__(self):
+        lint.LintPlugin.__init__(self)
+        self.namespace_prefixes = ['urn:mef:yang:' 'urn:mef:xid']
```

Comment:
You are missing a comma in the prefix list.

This would have been caught if you also provided some test cases...